services/py-api/src/service/participants_service.py

```diff
@@ -35,10 +37,45 @@ async def _create_participant_and_team_in_transaction(
             return participant
 
         return Ok((participant.ok_value, team.ok_value))
+    
+    async def _create_random_participant_in_transaction(
+            self, input_data: ParticipantRequestBody, session: Optional[AsyncIOMotorClientSession] = None
+    ) -> Ok[Participant] | Err[DuplicateEmailError | DuplicateTeamNameError | Exception]:
+        participant = await self._participant_repo.create(input_data, session, team_id=None)
```

Comment:
I think you can omit team_id from the arguments that you pass to create.

Reply:
Fixed in 833a26f627a807eaac502e53028d46fe1ca15419